pkg/status/combinedstatus-resolution.go

```diff
@@ -658,20 +658,74 @@ func refValToValue(val ref.Val) v1alpha1.Value {
 			Type:   v1alpha1.TypeString,
 			String: &v,
 		}
-	case int, int8, int16, int32, int64:
-		numStr := strconv.Itoa(int(v.(int64)))
+	case int:
```

Comment:
I see why casting an `int64` to an `int` is dangerous: https://go.dev/ref/spec#Numeric_types says that an `int` is either 32 or 64 bits.

I do not see why we need to write out separate cases for each of the five signed integer types. What would go wrong if the code said `case int, int8, int16, int32, int64:` and used `strconv.FormatInt(int64(v), 10)`?